client/blocks/reader-post-card/style.scss

```diff
@@ -209,12 +209,6 @@ $reader-post-card-breakpoint-small: "( max-width: 550px )";
 			color: $gray;
 		}
 
-		.reader-post-options-menu {
```

Comment:
@bluefuton Removed this block. It doesn't seem to be used elsewhere? 

Moved it over here instead: https://github.com/Automattic/wp-calypso/pull/17070/files#diff-f24f78e5492324b15efdb9dd3d9b0e6fR10